dashboard/app/models/concerns/partial_registration.rb

```diff
@@ -47,6 +47,12 @@ def self.persist_attributes(session, user)
     session[USER_ATTRIBUTES_SESSION_KEY] = user.attributes
   end
 
+  def self.cancel(session)
+    SignUpTracking.end_sign_up_tracking(session)
```

Comment:
I know we haven't spec'd it, but it might be helpful to send a metrics event for cancel right before we end sign-up tracking here.

Reply:
good call. i'll add a cancel event that gets logged for the experiment group (which is currently the only group that sees the "cancel" link)